shotover-proxy/example-configs/redis-cluster-dr/topology.yaml

```diff
@@ -19,10 +19,12 @@ chain_config:
           - QueryCounter:
               name: "DR chain"
           - RedisSinkCluster:
-              first_contact_points: [ "127.0.0.1:2120", "127.0.0.1:2121", "127.0.0.1:2122", "127.0.0.1:2123", "127.0.0.1:2124", "127.0.0.1:2125" ]
+              destination:
+                Hiding: [ "127.0.0.1:2120", "127.0.0.1:2121", "127.0.0.1:2122", "127.0.0.1:2123", "127.0.0.1:2124", "127.0.0.1:2125" ]
     - QueryCounter:
         name: "Main chain"
     - RedisSinkCluster:
-        first_contact_points: [ "127.0.0.1:2220", "127.0.0.1:2221", "127.0.0.1:2222", "127.0.0.1:2223", "127.0.0.1:2224", "127.0.0.1:2225" ]
+        destination:
+          Hiding: [ "127.0.0.1:2220", "127.0.0.1:2221", "127.0.0.1:2222", "127.0.0.1:2223", "127.0.0.1:2224", "127.0.0.1:2225" ]
```

Comment:
Hiding here as a keyword feels confusing (are we hiding these nodes or are we including these in the hidden cluster).